<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordConfirmation</class>
 <widget class="QDialog" name="PasswordConfirmation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-image: url(:/Icons/ProgramIcons/bg.png);
}

QLineEdit {
	font: 9pt &quot;Joan&quot;;
    border: 2px solid gray;
    border-radius: 10px;
	border-color: rgb(13, 54, 101);
     padding:4px 20px 4px 4px;
    background: white;
    selection-background-color: darkgray;
}

QPushButton{
font: bold 10pt  &quot;Heebo&quot;;
background-color: rgb(13, 54, 101);
color: rgb(233, 236, 237);
border-radius: 10px;
padding: 5px;
}
QPushButton::hover{
border-style: inset;
background-color:rgb(25, 104, 195);
color: rgb(213, 173, 18);

}
QPushButton::pressed{
border: 2px solid rgb(0, 170, 0);
}

QComboBox{
	font: 9pt &quot;Joan&quot;;
    border: 2px solid gray;
    border-radius: 10px;
	border-color: rgb(13, 54, 101);
     padding:4px 20px 4px 4px;
    background: white;
    selection-background-color: darkgray;
}
QComboBox::drop-down 
{
    border: 0px;
}
QComboBox::down-arrow{
   
	image: url(:/Icons/ProgramIcons/downarrow.png);
    width: 10px;
    height: 10px;
    }

QListView{
font: 9pt &quot;Joan&quot;;
    border: 2px solid gray;
    border-radius: 10px;
	border-color: rgb(13, 54, 101);
     padding:4px 20px 4px 4px;
    background: white;
    selection-background-color: darkgray;
}

QRadioButton::indicator::unchecked{ 
border: 1px solid; 
border-color: rgb(13, 54, 101);
border-radius: 9px;
border-width: 2px;
background-color: white; 
width: 15px; 
height: 15px; 
}

QRadioButton::indicator::checked{ 
border-color: rgb(13, 54, 101);
background-color:  rgb(213, 173, 18);
border: 1px solid; 
border-radius: 9px;
border-width: 2px;
width: 15px; 
height: 15px; 
}

QScrollBar:vertical {
border: none;
	background-color: rgb(208, 244, 255);
width: 14px;
margin: 15px 0 15px 0;
border-radius: 7px;
}


/* HANDLE BAR VERTICAL */
QScrollBar::handle:vertical {
	background-color: rgb(12, 12, 84);
min-height: 30px;
border-radius: 7px;
}
QScrollBar::handle:vertical:hover{
background-color:rgb(28, 114, 206)
}
QScrollBar::handle:vertical:pressed {
	background-color: rgb(8, 32, 59);
}


/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:vertical {
border: none;
background-color: rgb(213, 173, 18);
height: 15px;
border-top-left-radius: 7px;
border-top-right-radius: 7px;
subcontrol-position: top;
subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {
background-color: rgb(211, 213, 66);
}
QScrollBar::sub-line:vertical:pressed {
	background-color: rgb(139, 111, 12);
}


/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical {
border: none;
background-color: rgb(213, 173, 18);
height: 15px;
border-bottom-left-radius: 7px;
border-bottom-right-radius: 7px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {
background-color: rgb(211, 213, 66);
}
QScrollBar::add-line:vertical:pressed {
background-color: rgb(139, 111, 12);
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="30,70">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Brand Logo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Brand Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="UserFullName">
     <property name="text">
      <string>User Full Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="LabelContents">
     <property name="text">
      <string>Enter your current/new password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="PasswordLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="70,30">
     <item>
      <widget class="QPushButton" name="ConfirmPassword">
       <property name="text">
        <string>Confirm Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
